### PR TITLE
dead thoughts (starclan edition)

### PR DESCRIPTION
**Pull request by capillarys**
initially was only going to do newborns and medicine cats, but kind of went ham :P now have gotten new thoughts for a lot of dead cats (apps, deputy, med, kits, potats)!! praying nothing breaks </3

### CODE REVIEW

File: resources/dicts/thoughts/dead/starclan/newborn.json [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+        "id": "gen_dead_newborn",
         "thoughts": [
             "Mewls pitifully for milk",
-            "Tries to figure out all of the smells around them",
             "Snoozes peacefully",
-            "Tries to open their eyes, but it's all blurry"
+            "Wobbles around StarClan happily",
+            "Is drinking milk from a StarClan queen",
+            "Tries opening their eyes to see the 'stars' everyone is talking about",
+            "Is drinking milk from a StarClan queen",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/dead/starclan/newborn.json]: this is a duplicate thought

File: resources/dicts/thoughts/dead/starclan/newborn.json [hunk 2]
```diff
@@ -27,5 +33,20 @@
         "random_living_status": [
             "living"
         ]
+    },
+    {
+        "id": "gen_dead_newborn_to_dead_other",
+        "thoughts":[
+            "Is being watched by r_c",
+            "r_c is bathing them",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/dead/starclan/newborn.json]: "is being bathed by r_c" fits the thought format better

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

File: resources/dicts/thoughts/dead/starclan/newborn.json (hunk 2 above)
Comment (capillarys) [reply resources/dicts/thoughts/dead/starclan/newborn.json]: sorry! i copied it over from the ideas doc and didn't proofread, my bad

**Review by Ryos00 — APPROVED** [review]

**Comment by capillarys** [comment]
wait lemme add some more thoughts for the roles that didn't have much/any

**Pull request merged**